README.md

```diff
@@ -107,6 +107,23 @@ See the [getting started documentation](https://developer.amazon.com/docs/ask-to
 
 and many more ...
 
+----
+
+## FAQs
+
+- **How do I start Alexa Skills Kit Toolkit over SSH remote development?**
```

Comment:
I would rephrase this to `How do I setup and configure Alexa Skills Kit Toolkit extension when doing Visual Studio Code Remote Development using SSH?` ? or something of that sort? since this step is only for the first time someone tries to configure their extension and not needed on subsequent runs